src/services/posts/get-post.test.js

```diff
@@ -0,0 +1,149 @@
+const {expect} = require('chai')
+const db = require("../../db")
+const getPost = require("./get-post")
+const {fixtures} = require('../../../test/utils')
+
+
+describe('Get POST ', () => {
+
+	let user
+
+	before(async ()=>{
+		user = await fixtures.insertUser() 
+	})
+
+	describe('Gif', () => {
+		
+		let post
+
+		before(async ()=>{
+			post = await fixtures.insertPost({
+				userId : user.id , 
+				type : 'gif'
+			})
+			await fixtures.insertPostComment({
+				id : post.id,
+				userId : user.id,
+				type: 'gif'
+
+			})
+		})
+
+		it('should throw an error if Gif does not exist', async () => 
+		
+			expect(getPost({
+				id: 1234,
+				type: 'gif'}))
+				.to.be.rejectedWith('Gif does not exist')
+		)
+
+
+		it('should return a gif', async () => {
+
+			const insertedPost = await getPost({ 
+				id: post.id,
+				type : 'gif'
+		 })
+		 	
+			const result = await db.query(
+				`SELECT p.*, jsonb_agg(c.* ORDER BY c."createdAt" DESC) 
+				as comments FROM posts p 
+				LEFT JOIN comments c ON p.id = c."postId"
+				WHERE p.id=$1 AND p.type =$2
+				GROUP BY p.id;`,[ post.id, post.type])
+			const expectedPost = result.rows[0]
+			const {comments} = expectedPost
+			const comment = comments[0]
```

Comment:
for this, let's kindly compare the response from the service with what was inserted in the `before` block, similar to the api test

we usually want the function to match what we expect and replicating the same query in the service function itself might not necessarily be the ultimate confirmation we want